R/motif_enrichment_HOMER.R

```diff
@@ -95,6 +96,16 @@ dumpJaspar <- function(filename,
     # load PFMs and convert to HOMER format (base probabilties)
     requireNamespace(pkg)
     mdb <- utils::getFromNamespace(pkg, ns = pkg)
+    if (!methods::existsMethod(f = getMatrixSet, signature = list(x = class(mdb)))) {
```

Comment:
Do we want to specify the function like this: `f = TFBSTools::getMatrixSet`

Reply:
good point, I have added the namespace

Reply:
ah true, sounds good!